manifest.json

```diff
@@ -0,0 +1,144 @@
+{
+  "dependencies": {
+    "angular": {
+      "version": "1.4.7",
+      "path": "https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7",
+      "dest_target": "angularjs-",
+      "files": ["angular.js", "angular.min.js", "angular.min.js.map",
+        "angular-animate.js", "angular-animate.min.js", "angular-animate.min.js.map",
+        "angular-resource.js", "angular-resource.mini.js", "angular-resource.min.js.map",
+        "angular-route.js", "angular-route.mini.js", "angular-route.min.js.map",
+        "angular-sanitize.js", "angular-sanitize.mini.js", "angular-sanitize.min.js.map",
+        "angular-aria.js", "angular-aria.mini.js", "angular-aria.min.js.map"
+      ],
+      "jsFiles": ["angular.js", "angular-animate.js", "angular-resource.js",
```

Comment:
Suggest breaking after [

But what's the difference between files and jsFiles?